samples/vision_edge/edge_container_predict/automl_vision_edge_container_predict.py

```diff
@@ -18,18 +18,45 @@
 
 Examples:
     python automl_vision_edge_container_predict.py \
-    --image_file_path=./test.jpg --image_key=1 --port_number=8051
+    --image_file_path=./test.jpg --image_key=1 --port_number=8501
 
 """
 
 import argparse
 # [START automl_vision_edge_container_predict]
 import base64
+import cv2
 import io
 import json
 
 import requests
 
+def preprocess_image(image_file_path, max_width, max_height):
```

Comment:
This function is called `preprocess_image()` but it looks like the main thing it does is resize--is that correct? If so, I suggest renaming it.

Reply:
The current implementation mainly did read, resize and encoded it as string.
Will preprocess be a better name?